src/react.ts

```diff
@@ -1,7 +1,17 @@
-import { useCallback, useDebugValue, useEffect, useMemo, useRef } from 'react'
 import {
-  affectedToPathList,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  experimental_use,
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62189

It seems that the typing of `experimental_use` is added to the `@types/react`, and `ts-ignore` could be removed.

Reply:
Oh, yeah, I updated [there](https://github.com/jotai-labs/jotai-signal) and [there](https://github.com/jotai-labs/jotai-uncontrolled), but forgot here. Thanks.